package.json

```
@@ -1,13 +1,13 @@
 {
-  "name": "@react-three/cannon",
-  "version": "4.9.0",
-  "description": "physics based hooks for react-three-fiber",
+  "name": "@pmndrs/use-cannon",
+  "description": "root package for yarn workspaces monorepo",
```

This isn't a "yarn workspaces" project

```suggestion
  "description": "monorepo for @pmndrs/use-cannon",
```